lightly/cli/_helpers.py

```diff
@@ -65,3 +65,65 @@ def load_state_dict_from_url(url, map_location=None):
     # in this case downloading the pre-trained model was not possible
     # notify the user and return
     return {'state_dict': None}
+
+
+def _expand_batchnorm_weights(model_dict, state_dict, num_splits):
+    """Expands the weights of the BatchNorm2d to the size of SplitBatchNorm.
+
+    """
+    running_mean = 'running_mean'
+    running_var = 'running_var'
+
+    for key, item in model_dict.items():
+        # not batchnorm -> continue
+        if not running_mean in key and not running_var in key:
+            continue
+        
+        state = state_dict.get(key, None)
+        # not in dict -> continue
+        if state is None:
+            continue
+        # same shape -> continue
+        if item.shape == state.shape:
+            continue
+
+        # found running mean or running var with different shapes
+        state_dict[key] = state.repeat(num_splits)
+
+    return state_dict
+
+
+def _filter_state_dict(state_dict):
+    """Prevents unexpected key error when loading PyTorch-Lightning checkpoints.
+
+    Removes the "model." prefix from all keys in the state dictionary.
```

Comment:
instead of making this a doc maybe make the offset a context that denotes that it is removing this prefix?